### PR TITLE
Add React template

### PR DESCRIPTION
**Pull request by artem-kurchenko**
- A simple template with DevExtreme button and  Material Compact Theme
- All unnecessary styles are removed
- Strict Mode removed from index.js
- Readme.md updated to match the default pattern we have in the current examples

### CODE REVIEW

File: React/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  "name": "test-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.4",
+    "@testing-library/react": "^11.1.0",
+    "@testing-library/user-event": "^12.1.10",
+    "devextreme": "^20.2.5",
+    "devextreme-react": "^20.2.5",
+    "react": "^17.0.1",
```
Comment (LexDX) [inline React/package.json:11]: It is necessary to downgrade to 16, for now, bcs of the bug in our React integration. Also, a peer-dep error may occur

File: React/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
+  </head>
+  <body>
```
Comment (LexDX) [inline React/public/index.html:29]: dx-viewport ?

File: React/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
```
Comment (LexDX) [inline React/public/index.html:14]: I think we can remove all these comments

File: React/src/App.css [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.App {
```
Comment (LexDX) [inline React/src/App.css:1]: Do we really need anything there?

**Review by LexDX — CHANGES_REQUESTED** [review]
We need to get rid of yarn.lock. The farm just does not use yarn. Also, maybe we can remove basic comments.

File: React/src/App.css (hunk 4 above)
Comment (artem-kurchenko) [reply React/src/App.css:1]: yep

**Review by LexDX — COMMENTED** [review]
lgtm

**Pull request merged**